src/util/log.js

```diff
@@ -0,0 +1,18 @@
+import anylogger from 'anylogger';
+import 'anylogger-console';
+/**
+* @desc Determines whether or not to log based on a url param. Takes description as a parameter and returns log.
+* @param {string} description - The description that should go in the log.
+**/
+export function sendLog(parentFunc, description, slotName) {
```

Comment:
Please write tests for sendLog:
<img width="617" alt="Screen Shot 2021-01-20 at 2 31 31 PM" src="https://user-images.githubusercontent.com/2664083/105225218-4046c300-5b2c-11eb-821c-bd9e7884ec2c.png">